### DISCUSSION TITLE
Certificate location

### DISCUSSION THREAD

**Discussion by jameshiscott** [mockoon/mockoon#1388] [Help] [answered]
Hi, This is a super basic question but I simply cant find or workout the answer. I am running mockoon via Docker:

docker run -d -p 3001:3001 mockoon/cli:latest -d https://mygiturl/myfile.json -p 3001

I have enabled TLS in the file and have the certs (certificate.crt, ca_bundle.crt and private.key) but my question is
1) what do i put in the mockoon settings page for the location
2) and very related, where do i need to put the certs on my ubuntu host?

I am sorry this is such a basic question but i have tried everything i can think of.

**Comment by 255kb** [comment, marked as answer]
Hi, most file paths in Mockoon support both absolute paths and relative paths. If you use an absolute path, you will have to put the file in the same spot on all the machines. However, if you provide a relative path, it is automatically resolved from the data file you are loading. ("myfile.json").
In you example, you are loading a data file using a URL, so, the relative path trick will not work. I suggest that you first download this file using something like `wget https://mygiturl/myfile.json`, put the cert files in the same folder, and then run the CLI using `docker run -d -p 3001:3001 mockoon/cli:latest -d ./myfile.json -p 3001`. 

And in your TLS setup, you use relative paths: 
![image](https://github.com/mockoon/mockoon/assets/7489814/fee79cb0-47b5-4907-b94a-d27a1bd5c48c)

**Reply by jameshiscott** [reply]
Perfect this worked thanks. I didn't know we could use relative paths but thanks for the full overview

**Comment by chrlyck** [comment]
Hi, I have sort of the same issue, but am building using dockerize.

When I start the image, it stops immediately and I get:

{"app":"mockoon-server","environmentName":"x","environmentUUID":"6833fc70-c159-40ce-a5b4-6608229d8892","level":"error","message":"An unknown server error occured: unsupported","timestamp":"2025-03-10T07:38:39.978Z"}
Error: An unknown server error occured: unsupported

Any way I can get more information?

**Reply by 255kb** [reply]
"Unsupported" is the message from the server error (Node.js or Express) that was thrown. It's indeed not very useful and there is no way to know where it comes from. 
Can you create a minimal reproducible example?
